src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -565,6 +566,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
                 End While
                 If CanGet(Here) AndAlso IsSingleQuote(Peek(Here)) Then
                     If CheckFeatureAvailability(Feature.CommentsAfterLineContinuation) Then
+                        ' You only get here if running V16 and have a comment
```

Comment:
@paul1956 as mentioned already, this is *not* what we want.  We want to parse the same way (except for diagnostics) regardless of what version you are on.

So, we will always lex out ```_ ' comment``` trivia in any language version.  However, if you are below v16, then we'll report a diagnostic if you use this construct.

Reply:
Thanks I now understand the feedback and have just posted a fix, plus new tests.